.github/workflows/update-snapshots.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         run: pnpm -r build
 
       - name: Purge existing snapshots
-        run: rm packages/**/snapshots/**/*.png
+        run: rm packages/**/**/snapshots/**/*.png
```

Comment:
Das ist spoky ... `/**/` heißt doch schon `n`-viele Verzeichnisse?!

Reply:
Es kommt wohl auf die Umgebung an. In einer normalen Bash funktioniert es nicht rekursiv, nein.
Kann aber mit `shopt -s globstar` aktiviert werden.

Reply:
Meine Güte, ich habe nie anders gearbeitet. Danke für den Hint.

Reply:
Ps: das war aber vorher schon nur mit einmal **